packages/calcite-components/src/components/combobox/assets/combobox/t9n/messages.json

```diff
@@ -1,4 +1,6 @@
 {
+  "all": "All",
```

Comment:
You need to also update `messages_en.json` (see https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/conventions/Internationalization.md#t9ncomponent-pattern).

We have an NPM script to help with this (`util:sync-t9n-en-bundles`), but it appears it needs to be broken. Probably needs to be updated to account for the monorepo structure).

Sidebar: @anveshmekala we need to add a note about updating strings to the doc.